draft-ietf-quic-multipath.md

```diff
@@ -526,36 +526,36 @@ peer during the address validation procedure. According to
 is to not send a PATH_RESPONSE in response to the peer's PATH_CHALLENGE.
 
 
-### Allocating, Consuming and Retiring Connection IDs {#consume-retire-cid}
-
-Each connection ID is associated with a Path ID.
-The Path Identifier 0 indicates the initial path of the connection.
-Endpoints SHOULD issue at least one unused connection ID with an unused path identifier.
-
-An endpoint maintains a set of connection IDs received from its peer for each path,
-any of which it can use when sending packets, as specified in {{Section 5.1 of QUIC-TRANSPORT}}.
-In the multipath extension, each connection ID belongs to one path specified by
-the Path Identifier field of MP_NEW_CONNECTION_ID frame in {{mp-new-conn-id-frame}}.
-The connection IDs used during the handshake belong to the initial path
-with Path ID 0. Usually, it is desired to provide at least one more connection ID for
-for all used path and at least one for an unused Path ID, to allow for migration and
-opening of new paths.
-
-If the client has used all the allocated connection IDs, it is supposed to retire
+### Allocating, Consuming, and Retiring Connection IDs {#consume-retire-cid}
+
+With the multipath extension, each connection ID is associated with one path
+that is identified by the Path ID as specified in the Path Identifier field of
+the MP_NEW_CONNECTION_ID frame {{mp-new-conn-id-frame}}.
+The Path ID 0 indicates the initial path of the connection.
+Respectively, the connection IDs used during the handshake belong to the initial path
+with Path ID 0.
+The MP_NEW_CONNECTION_ID frame is used to issue new connection IDs for all paths.
+Endpoints SHOULD issue at least one connection ID for an unused path identifier
+to allow new paths to be opened.
+
+Each endpoint maintains the set of connection IDs received from its peer for each path,
+any of which it can use when sending packets on that path; see also {{Section 5.1 of QUIC-TRANSPORT}}.
+Usually, it is desired to provide at least one additional connection ID for
+all used paths, to allow for migration.
+
+{{Section 5.1.2. of QUIC-TRANSPORT}} specifies the retirement of connection IDs.
+In order to identify a connection ID correctly when the multipath extension is used,
+endpoints have to use the MP_RETIRE_CONNECTION_ID frame instead
+of the RETIRE_CONNECTION_ID frame to indicate the respective Path ID together with the
+connection ID sequence number, at least for all paths with a Path ID other than 0.
```

Comment:
```suggestion
connection ID sequence number, at least for all paths with a Path ID other than 0.
To keep compatible with RFC9000, endpoints MUST treat RETIRE_CONNECTION_ID frame as 
used for initial Path (with Path ID 0), and it doesn't cause extra error.
```

Reply:
We don't need to use normative language here because this specification does not invalidate any RFC9000 frames. Therefore NEW/RETIRE_CID frames must be processed. The only thing we need to say is that the path ID for these CIDs is 0. That also doesn't need normative language either because that just a fact that is needed to be specified to implement this extension but not something for the endpoints to act on in a specific way.